shared/test/showcase/test_helper.rb

```diff
@@ -77,13 +101,16 @@ def new_compliance_rest_client
   @showcase_id = begin
     server_id = nil
     if ENV['CI'].nil?
-      puts "Starting showcase server..." if ENV["VERBOSE"]
-      server_id, status = Open3.capture2 "docker run --rm -d -p 7469:7469/tcp -p 7469:7469/udp "\
-        "gcr.io/gapic-images/gapic-showcase:0.25.0"
-      raise "failed to start showcase" unless status.exitstatus.zero?
-
-      server_id.chop!
-      puts "Started with container id: #{server_id}" if ENV["VERBOSE"]
+      unless gapic_showcase_running?
+        log_file = "/tmp/gapic-showcase.log"
```

Comment:
Recommend using `Dir.tmpdir`, or better yet `Dir.mktmpdir` to create a new tmp dir private to this test run, instead of hardcoding the path `"/tmp"`. (You'll first need to `require "tmpdir"` of course.) Here and other places this directory is used below.

Reply:
`tmpdir` is already there -- using it now! Thanks